### ISSUE TITLE
Build instructions

### ISSUE THREAD

**Issue by daic11** [coll4ctg43/simplex-chat-arm64#1]
Hello!

Have you build simplex-chat.xz with "cabal build" command?
What the directory the final file will be located?

I try to build myself rpm for fedora.
Thank you!

**Comment by coll4ctg43** [comment]
no, I used podman which simulates docker. Inside the Dockerfile then there are all the commands and scripts to compile and I think also cabal. The file is xz because I had to compress it otherwise it was too big for github. If you use docker you pass the directory to it as input. The instructions for using docker are here
https://github.com/simplex-chat/simplex-chat/blob/stable/docs/CLI.md#using-docker

**Comment by daic11** [comment]
Need same instructions but for GUI Desktop
https://github.com/simplex-chat/simplex-chat/blob/stable/docs/CLI.md#using-docker

**Comment by coll4ctg43** [comment]
is not a bug and does not compete with this repository, you have to ask the developers of https://github.com/simplex-chat/simplex-chat
